client/src/components/NationwideMap.vue

```diff
@@ -323,20 +323,20 @@ export default defineComponent({
     },
     // Listens to query param to toggle layers.
     routerLayer: function(newLayer: string) {
-      if (newLayer != null) {
+      if (newLayer != null && this.map?.isStyleLoaded()) {
```

Comment:
Don't even need this watcher since app.vue dispatches an event to change the map's current layer

Reply:
Removed!